docs/src/domains/cdn/fastly.md

```diff
@@ -1,55 +1,57 @@
 ---
-title: "Fastly"
+title: "Configure your Fastly CDN"
+sidebarTitle: "Fastly"
 weight: 1
-description: In some cases you may want to opt to use a CDN such as Fastly rather than the Platform.sh router's cache. Using a CDN can offer a better time-to-first-byte for cached content across a wider geographic region at the cost of the CDN service.
+description: Learn how to configure your Fastly CDN.
+aliases:
+  - "/golive/steps/fastly.html"
 ---
 
-{{% description %}}
+You can [use a CDN](./_index.md) to deliver your site's content to users more quickly.
+To set up a Fastly CDN on a Grid project,
+follow the instructions on this page.
 
-A Fastly CDN is included for Dedicated projects.
-Platform.sh doesn't offer an integrated CDN on self-service Grid projects at this time,
-but it is a common choice for customers to self-configure.
+**Dedicated users**: Fastly CDNs included in Dedicated projects are set up by Platform.sh. If you need help with your Fastly CDN settings,
```

Comment:
:lady_beetle: This is the only instance of referring to `Dedicated users`. I think it'd be best to talk about Dedicated plans or projects or the like.

:hammer_and_wrench: This might be the first time someone sees that a Fastly CDN specifically is included in a Dedicated plan, so I might rephrase that part. Especially to emphasize that they're fully managed by Platform.sh (not just set up, but afterwards too).
```suggestion
**Dedicated plans** come with a Fastly CDN fully managed by Platform.sh.
To modify any settings for a managed Fastly CDN,
```